frontend/hub/namespaces/HubNamespaceForm.tsx

```diff
@@ -59,10 +59,10 @@ export function EditHubNamespace() {
     data: namespace,
     error,
     refresh,
-  } = useGet<HubNamespace>(hubAPI`/_ui/v1/namespaces/${name ?? ''}/`);
-  const patchRequest = usePatchRequest<HubNamespace, HubNamespace>();
+  } = useGet<HubNamespace>(hubAPI`/_ui/v1/my-namespaces/${name ?? ''}/`);
+  const putRequest = usePutRequest<HubNamespace, HubNamespace>();
   const onSubmit: PageFormSubmitHandler<HubNamespace> = async (namespace) => {
-    await patchRequest(hubAPI`/_ui/v1/namespaces/`, namespace);
+    await putRequest(hubAPI`/_ui/v1/my-namespaces/${name ?? ''}/`, namespace);
     navigate(-1);
```

Comment:
@jamestalton @vishsanghishetty could you take a look at this use of `navigate(-1)` on the ns edit form? If you _edit namespace_ in the ui, you can see that navigate only takes you back to the previous page and does not trigger a reload, which we need for the fields to be updated. Do you know of anything we can replace this with that will trigger reload? Thank you~